src/equations/scalar_diffusion_2d.jl

```diff
@@ -0,0 +1,10 @@
+struct ScalarDiffusion2D{T} <: AbstractParabolicEquations{2, 1}
+  diffusivity::T
+end
+
+# no orientation here since the flux is vector-valued
```

Comment:
I don't think it would be applicable for systems. For example, with CNS the viscous flux still depends only `u` and `u_grad`, it just combines gradients of different solution components as well.